.github/workflows/test-sigstore.yml

```diff
@@ -11,6 +11,7 @@ permissions: {}
 jobs:
   test-sigstore:
     runs-on: ubuntu-latest
+    if: github.repository_owner == 'secure-systems-lab' # only run upstream
```

Comment:
@jku: I think we want this line in the kms workflow as well, even more than here actually. Because kms just fails in a fork that does not have GCP configured. Usually it even fails two times in a fork, once for the missing GCP config, and once for trying to submit an issue, which are disabled by default. It's not a big deal, but adds an ❌ 

Sigstore tests should pass in a fork, I just restricted the workflow to not unnecessarily spam sigstore certificate logs.